lib/tty0tty.ex

```diff
@@ -78,12 +78,26 @@ defmodule TTY0TTY do
     cmd = [tty0tty, [dev_path, twin], opts]
 
     case DynamicSupervisor.start_child(__MODULE__, {MuonTrap.Daemon, cmd}) do
-      {:error, {:already_started, _p}} -> :ok
-      {:ok, _} -> :ok
-      result -> raise "Failed to open tty0tty! - #{inspect(result)}"
+      {:error, {:already_started, _p}} ->
+        :ok
+
+      {:ok, _} ->
+        wait_for_paths_to_exist(dev_path, twin)
+        :ok
+
+      result ->
+        raise "Failed to open tty0tty! - #{inspect(result)}"
     end
   end
 
+  defp wait_for_paths_to_exist(path, twin) do
+    with false <- ensure_paths_exist(path, twin), do: wait_for_paths_to_exist(path, twin)
```

Comment:
I like this! I am also a little concerned that if something happens with the file or PTY process between supervisor start and here, then it would never exit the loop. I feel like we might want a way to break out after some time.

Another option I was thinking would be to use the [`log_transform` option with `MuonTrap.Daemon`](https://github.com/fhunleth/muontrap/blob/main/lib/muontrap/daemon.ex#L145) and have a little function that is just blocking until we get the [log line that the ports have been created](https://github.com/jjcarstens/tty0tty/blob/main/c_src/tty0tty.c#L194-L197) - though I'm not sure that is any better, but an idea